src/collection.js

```diff
@@ -494,6 +494,21 @@ class Collection {
     return this
   }
 
+  /**
+   * Returns median of the current collection
+   *
+   * @param {}
+   *
+   * @returns {Number}
+   */
+  async median () {
+    const arr = (await this.sort()).items
+    const mid = Math.floor(arr.length / 2)
+    return arr.length % 2 !== 0

```

Comment:
Same here, please use `this.size()` instead of `arr.length`:

```suggestion
    return this.size() % 2 !== 0
```

Reply:
Done. Please Check